docs/organizations/overview.mdx

```diff
@@ -74,20 +74,12 @@ Membership requests are requests from users who want to join an organization. A
 
 ## Active organization
 
-There are two types of accounts that a single user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.
+By default, when a user signs in to a Clerk-powered application, there is no active organization set. Even if they are a member of only one organization, they must explicitly set it as active.
```

Comment:
nit: If a user had signed in, become active in an organization and then signs in again they would still be active in that organization.

Maybe "By default, when a user initially signs up or in to a Clerk-powered application..."?

I get the confusion of the customer who initially reported this page, but I feel like the information that is being removed is good but just poorly worded and confusing. @alexisintech -- maybe we could chat Monday afternoon for 15 minutes and see what we can come up with?

Reply:
This is a good suggestion - adding "initially" does clarify a bit here. We could also shift the framing a little from "there's no active organization set" to "the user has a 'personal workspace' that is set to active, even if they are a member of one or more organizations"